students/chylak_malgorzata/lesson_01_basics/fractional_part.py

```diff
@@ -0,0 +1,7 @@
+import math
+
+n = float(input())  # float, because there's a number with a comma
+print(math.modf(n))
+print('0.'.join(str(n).split('.')[1]))
+frac = n - int(n)
+print("%.2f" % frac)
```

Comment:
Which of this three prints should I treat as a proper answer :)? The last line is printing almost good result. Change that print to print(frac) and comment or delete rest of the prints